src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -22,9 +23,10 @@
             + "Example: " + COMMAND_WORD
             + " " + PREFIX_NAME + "Khong Guan "
             + " " + PREFIX_NAME + "Swee Choon"
-            + " " + PREFIX_TASK_DESC + "Scheduled 3rd Pfizer shot"
+            + " " + PREFIX_TASK_DESC + "Weekly Taiji"
             + " " + PREFIX_TASK_DATE + "2021-10-10"
-            + " " + PREFIX_TASK_TIME + "14:30";
+            + " " + PREFIX_TASK_TIME + "14:30"
+            + " " + PREFIX_TASK_RECURRING + "week";
```

Comment:
Perhaps you can include the `[ ]` for this field to indicate this is optional, based on the current implementation